src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/CompareAttribute.cs

```diff
@@ -60,8 +60,6 @@ public CompareAttribute(string otherProperty) : base(SR.CompareAttribute_MustMat
             var display = attributes.OfType<DisplayAttribute>().FirstOrDefault();
             if (display != null)
             {
-                // TODO-NULLABLE: This will return null if [DisplayName] is specified but no Name has been defined - probably a bug.
```

Comment:
while this might be true I've removed it because it doesn't affect any public API in the end